node/parallel/src/service.rs

```diff
@@ -161,7 +161,7 @@ async fn start_node_impl<RB>(
     polkadot_config: Configuration,
     id: ParaId,
     validator: bool,
-    rpc_ext_builder: RB,
+    _rpc_ext_builder: RB,
```

Comment:
IMHO, we should create a `rpc_ext_builder` instead of  ignoring it

Reply:
rpc_ext_builder depends on transaction_pool which will be created in the start_node_impl function, so I removed this parameter instead.